examplebroker/broker.go

```diff
@@ -586,8 +586,8 @@ func (b *Broker) handleIsAuthenticated(ctx context.Context, sessionInfo sessionI
 		exampleUsersMu.Lock()
 		defer exampleUsersMu.Unlock()
 
-		if challenge != "newpass" {
-			return AuthRetry, `{"message": "new password does not match criteria: must be newpass"}`, nil
+		if challenge != "authd2404" {
```

Comment:
I think pwquality stroke here :) (I was first "why did you change this?")

Reply:
Yeah, in my tests I had to use `minlen=6` (note that pwquality's min length is actually `N+1` :scream:).